GeoLib/Raster.cpp

```diff
@@ -148,9 +148,7 @@ double Raster::interpolateValueAtPoint(MathLib::Point3d const& pnt) const
 	for (unsigned j=0; j<4; ++j)
 	{
 		// check if neighbour pixel is still on the raster, otherwise substitute a no data value
-		if ( (xIdx + x_nb[j]) < 0 ||
-		     (yIdx + y_nb[j]) < 0 ||
```

Comment:
why did you remove the check if the calculated index is < 0?
that was precisely why the programme had crashed before?!